src/repositories/images/interfaces/imagesController.ts

```diff
@@ -0,0 +1,16 @@
+import { IImagesRepository } from './imagesRepository'
+
+type typeImage = {
```

Comment:
Зачем писать type в названии если и так понятно что это тип ?

```suggestion
interface Image {
```